src/ziggurat/kafka_consumer/consumer_driver.clj

```diff
@@ -43,7 +45,7 @@
 (defn- stop-consumers [consumer-groups]
   (do (log/info "stopping consumers")
       (doseq [[topic-entity consumers] consumer-groups]
-        (log/info "Stopping threads for consumer group: " topic-entity)
+        (clog/info {:consumer-group topic-entity} (str "Stopping threads for consumer group: " topic-entity))
```

Comment:
Even though `:consumer-group` is technically correct, people are used to topic-entity. Should we used that?

WDYT?

Reply:
log says consumer group while stopping consumers, so we went with it

Reply:
This should be a standard key and should be applied to all the logs from the streaming part of the application.

Reply:
Yeah. So, :consumer-group works right?